OpenMEEG/libs/OpenMEEGMaths/include/linop.h

```diff
@@ -51,6 +52,14 @@ namespace OpenMEEG {
         struct OPENMEEGMATHS_EXPORT MathsIO;
     }
 
+    // to properly convert a size_t int to an int
+    inline OPENMEEGMATHS_EXPORT int sizet_to_int(const size_t& num)
+    {
+        int num_out = static_cast<int>(num);
+        om_assert(num_out >= 0);
+        return num_out;
+    }
+
     class OPENMEEGMATHS_EXPORT LinOpInfo {
 
```

Comment:
or maybe I do a template function to return an unsigned also ? (rarely needed... only in mesh.{h,cpp})